give/src/App.js

```diff
@@ -70,17 +70,16 @@ const Nav = () => {
   );
 };
 
-const Home = () => {
-  return <p>Hello world!</p>;
-};
+// const Home = () => {
+//   return <p>Hello world!</p>;
+// };
```

Comment:
Are you planning on coming back to this at some point? Or was it just to show something?

Reply:
I forgot to delete it once I verified what I was replacing it with worked.